assets/handleKeyDownEvent.js

```diff
@@ -0,0 +1,11 @@
+import {displayParagraphs} from "./displayText.js";
+import {preventSpaceBar} from "./preventDefault.js";
+
+export function handleKeyDownEvent(event, paragraphWithText, statsText, todayStatsText, charArray, charIndex, seconds, userKeyTypeCount, userMistakesCount, amountOfSets) {
```

Comment:
when a function requires so many arguments, it can make sense to only have one argument which is an object. this way you can't mess up the order

Reply:
Okay I will use an object in future